driver/example_test.go

```diff
@@ -27,11 +27,11 @@ func ExampleDriver() {
 	_ = c.CreateDatabase(ctx, "db1", &DatabaseOptions{})
 
 	_ = c.CreateCollection(ctx, "db1", "coll1",
-		Schema(`{ "properties": { "F1": { "type": "string" }, "F2": { "type": "string" } }, "primary_key": ["F1"] }`), &CollectionOptions{})
+		Schema(`{ "properties": { "F1": { "type": "string" }, "F2": { "type": "string" } }, "primary_key": ["F1"] }`))
 
-	_, _ = c.Insert(ctx, "db1", "c1", []Document{Document(`{"F1":"V1"}`)}, &InsertOptions{})
+	_, _ = c.Insert(ctx, "db1", "c1", []Document{Document(`{"F1":"V1"}`)})
 
-	it, _ := c.Read(ctx, "db1", "c1", Filter(`{"F1":"V1"}`), &ReadOptions{})
+	it, _ := c.Read(ctx, "db1", "c1", Filter(`{"F1":"V1"}`), Fields(`{}`))
```

Comment:
No need to set fields?

Reply:
Fields is a mandatory parameter.
`{}` means read all fields

Reply:
in the client API? or server side? Server side if fields key is not present that means read all.